crates/events/src/extend.rs

```diff
@@ -517,6 +518,20 @@ impl EventAttributeEntry<'static> for MaspTxBatchRefs {
     }
 }
 
+/// Extend an [`Event`] with data sections for IBC shielding transfer.
+pub struct IbcTxBatchRefs(pub IbcTxDataRefs);
```

Comment:
Could we rename to something slightly different? This is used to notify the presence of masp over ibc transactions, not ibc txs in general

Reply:
Agree. Renamed to `IbcMaspTxBatchRefs`: 45d2798